### DISCUSSION TITLE
Make arena honest and great again(suggestions)

### DISCUSSION THREAD

**Discussion by Licse** [chromiecraft/chromiecraft#4912] [Suggestions]
I would like to suggest some easy and possibile answers for arena's disturbing and common problems.

1. As we all know 3v3 bracket isn't and won't be active enough to achieve tier 2 wep/shoulders(2.3k rating 3v3 only required) on a low population server. It creates and have been created on almost all known p servers(even the bigger ones) the win-trade issue in order to get those items. it is the only way to get the req rating and its very ez to trade-win  when rarely other teams quing, so u will always get the wanted win-trade opponents.
-Solution already tested and exists: 
Solo Q 3v3. 
It's great, it's fun, it brings papers from other servers and it's solving an annoying issue and also motivates not hard-core or simply newbies to start and explore the pvp aspect for the first times.  
Of course it will need its balancing and everything, but totally worth it when done  correctly.

2. Racial skills in arena, famous problem, that disturbes the inner peace of many pvpers who actually want to pvp equally while conserve the aestetics of a chosen faction or race.
-Solution option 1#:  simply disattivate any racial upon entrance in arena, keeps everything simple and real.
-Solution option 2#: monitize the option (with chromie points) to change the racials to a specific race without changing the look. 
Already scripted and has been written by CC player. Link : https://github.com/azerothcore/mod-racial-trait-swap

Hopefully those issues will get the needed attention from above.

**Comment by acidmanifesto** [comment]
Just fyi. I am not a CC player. Just a Custom Developer on various cores.
i mean. i support CC if they ever decided to use the racial swap
just need to keep in mind, i do not play on servers, i am not a pvper
i do not know the pro's and con's with such racial swapping
but i will say research took about a month with every possible combination per class and race
so the info is accurate
also. the racial swap was based on 335a
so i am not entirely certain what they got with the progression mod in play, if that would affect anything

**Comment by Cooties-cc** [comment]
Could a solution to problem 1 be to just remove rating requirements, and slightly increase item costs? Similar to how they were doing previous gear with honor points. Those who play more, or get a higher team rating, will in turn earn more, so they can obtain items at a faster rate. However those who are slower/lower will only obtain them at a slower rate, instead of being outright locked from the items? Solo ques seem good too, only if they go by a backend MMR system. No one likes to solo que only to see 1 team got randomly super stacked and you get curb stomped. Personal MMR could help ease the solo matchmaking process so it plays out more balanced. 

Problem 2 I always sided with racial removal (in higher skill pvp like arena, bg's are mostly w/e). I've always wondered later in the game design, when they stopped caring mostly about personal RPG aspects, to not have baseline PvE racials (think like underwater breathing, or passive non-helpful combat ones), and allowing people to pick PvP racials. It should have been built into the talent system in a way, where you could pick your class design. Opponents to this idea generally say it plays into class homogeneity, where if people can just pick what's best, every class will play the same. Problem with PvP I always saw is that racials were that slight edge advantage when each character is geared the exact same at the same playing level. However I'm against monetizing the "pick a racial" feature, as some may say that's pay2win, since not everyone could afford to pay money to get the swap. Perhaps ingame gold options, like with buying glyphs and such.

**Reply by Licse** [reply]
1. Taking down arena gear requirments (rating)kills all motivation to do arena at all.
It's not same ream as 3s solo q will shuffle, u can't q as party. So it will construct random parties of 3s. 
-and anyways its better same teams than no teams.

2. It's not pay2win if it's only aestetics that change. Racials remain as u chose them initially.

**Comment by balleny** [comment]
please, try to keep the discussions together
https://github.com/chromiecraft/chromiecraft/discussions/4161
